haystack/components/routers/metadata_router.py

```diff
@@ -7,59 +7,78 @@
-                            ],
-                        },
-                        "edge_4": {
-                            "operator": "AND",
-                            "conditions": [
-                                {"field": "meta.created_at", "operator": ">=", "value": "2023-10-01"},
-                                {"field": "meta.created_at", "operator": "<", "value": "2024-01-01"},
-                            ],
-                        },
-                      }
-                      ```
+        :param rules: A dictionary of rules that specify which output connection to route a document to based on its metadata.
+            The keys of the dictionary are the names of the output connections, and the values are dictionaries that
+            follow the [format of filtering expressions in Haystack](https://docs.haystack.deepset.ai/v2.0/docs/metadata-filtering).
+            For example:
+            ```python
+            {
+            "edge_1": {
+                "operator": "AND",
+                "conditions": [
+                    {"field": "meta.created_at", "operator": ">=", "value": "2023-01-01"},
+                    {"field": "meta.created_at", "operator": "<", "value": "2023-04-01"},
+                ],
+            },
+            "edge_2": {
+                "operator": "AND",
+                "conditions": [
+                    {"field": "meta.created_at", "operator": ">=", "value": "2023-04-01"},
+                    {"field": "meta.created_at", "operator": "<", "value": "2023-07-01"},
+                ],
+            },
+            "edge_3": {
+                "operator": "AND",
+                "conditions": [
+                    {"field": "meta.created_at", "operator": ">=", "value": "2023-07-01"},
+                    {"field": "meta.created_at", "operator": "<", "value": "2023-10-01"},
+                ],
+            },
+            "edge_4": {
+                "operator": "AND",
+                "conditions": [
+                    {"field": "meta.created_at", "operator": ">=", "value": "2023-10-01"},
+                    {"field": "meta.created_at", "operator": "<", "value": "2024-01-01"},
+                ],
+            },
+            }
+            ```
         """
         self.rules = rules
         component.set_output_types(self, unmatched=List[Document], **{edge: List[Document] for edge in rules})
 
     def run(self, documents: List[Document]):
         """
-        Run the MetadataRouter. This method routes the documents to different edges based on their fields content and
-        the rules specified during initialization. If a document does not match any of the rules, it is routed to
-        a connection named "unmatched".
+        This method routes the documents to different edges based on their fields content and the rules specified during initialization.
+        If a document does not match any of the rules, it is routed to a connection named "unmatched".
```

Comment:
```suggestion
        Route the documents to different edges based on their fields content and the rules specified during initialization.
        If a document does not match any of the rules, it is routed to a connection named "unmatched".
```
Let's use the imperative mood for docstrings.